cypress/e2e/signIn.cy.js

```diff
@@ -2,6 +2,24 @@
 
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
+    cy.visit('https://conduit.mate.academy/user/login');
 
+    cy.get('h1')
+      .should('contain.text', 'Sign in');
+
+    // email entering
+    cy.get('#email')
+      .type('bybeg@mailinator.com');
+
+    // password entering
+    cy.get('#password')
+      .type('Pa$$w0rd!');
+
+    cy.get('.btn')
+      .click();
+
+    cy.get(':nth-child(4) > .nav-link')
```

Comment:
The selector ':nth-child(4) > .nav-link' is brittle and can easily break with minor changes to the page structure. Consider using more robust selectors or add data-cy attributes to your elements.